cmr2_api/settings.py

```diff
@@ -45,6 +45,7 @@
     'rest_framework_gis',
     'django_filters',
     'corsheaders',
+    'django.contrib.gis',
```

Comment:
Check position, it might be on `django.contrib...` imports.

Reply:
Adjusted.